### PR TITLE
Added parsing logic if multiple protocols are supplied

### PR DESCRIPTION
**Pull request by norlandrhagen**
https://github.com/norlandrhagen/pangeo-forge-recipes/pull/new/protocol

### CODE REVIEW

**Comment by norlandrhagen** [comment]
Totally see how `minio` would be helpful! For now I just added a simple test, but it would be nice to include that in the future todo.

File: pyproject.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ dependencies = [
     "cftime",
     "dask >= 2021.11.2",
     "fastparquet",
-    "fsspec[http] >= 2023.4.0",
+    "fsspec[http,s3] >= 2023.4.0",
```
Comment (cisaacstern) [inline pyproject.toml:32]: Do we need `s3` here if we're including `s3fs` in dev dependencies below?

**Comment by cisaacstern** [comment]
Probably need to pass `anon=True` to workaround the missing credentials error being thrown in the tests. 🙂

**Comment by cisaacstern** [comment]
LGTM! Thanks so much @norlandrhagen!

**Pull request merged**